linux_os/guide/system/network/network-nftables/set_nftables_loopback_traffic/ansible/shared.yml

```diff
@@ -0,0 +1,38 @@
+# platform = multi_platform_sle
+
+- name: Implement Loopback Rules
+  ansible.builtin.command: nft add rule inet filter input iif lo accept
+
+- name: Create Rule to Drop Input IP Address from Loopback
+  ansible.builtin.command: nft insert rule inet filter input ip saddr 127.0.0.0/8 counter drop
+
+- name: Check if IPv6 is Disabled in grub Configuration
+  ansible.builtin.shell: |
+    [ -n "$passing" ] && passing=""
```

Comment:
It doesn't seem necessary to set `passing` to false or true here. You can simply use the `ipv6_status` result code in the last tasks.